src/renderers/dom/shared/__tests__/ReactRenderDocument-test.js

```diff
@@ -133,23 +131,28 @@ describe('rendering React components at document', () => {
     }
 
     var markup = ReactDOMServer.renderToString(<Component />);
-    testDocument = getTestDocument(markup);
+    var testDocument = getTestDocument(markup);
 
     ReactDOM.render(<Component />, testDocument);
 
     expect(testDocument.body.innerHTML).toBe('Hello world');
 
     // Reactive update
-    expect(function() {
+    if (ReactDOMFeatureFlags.useFiber) {
+      // This works but is probably a bad idea.
```

Comment:
Don't suppose you remember what you meant by this comment, @sebmarkbage?

Reply:
See UNMOUNT_INVARIANT_MESSAGE at the top of the file. Browsers don't like when you unmount the `html` host node, or at least some didn't when we added that invariant to Stack.

Reply:
hey Sophie 👋 

Thanks! I misread Seb's comment as "this works but isn't a good way to test" rather than "this works but we don't encourage it in production"

I follow now